### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by DO1JLR**
fixed all linting warnings and add meta for all roles

### CODE REVIEW

**Comment by DO1JLR** [comment]
ping @JanKoppe 
Ich hab endlich bischen was hieran gemacht :wink:

File: roles/docker/meta/main.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+galaxy_info:
+  role_name: docker
+  author: Jan Koppe <post@jankoppe.de>
+  description: iAdd repo and install docker
```
Comment (JanKoppe) [inline roles/docker/meta/main.yaml]: typo in "iAdd"

File: roles/grafana/meta/main.yaml [hunk 2]
```diff
@@ -1,5 +1,24 @@
+---
+galaxy_info:
+  role_name: grafana
+  author: Jan Koppe <post@jankoppe.de>
+  description: Install grafana and prometheus
```
Comment (JanKoppe) [inline roles/grafana/meta/main.yaml]: This role does not install prometheus.

File: roles/srs_edge/meta/main.yaml [hunk 3]
```diff
@@ -1,3 +1,21 @@
 ---
+galaxy_info:
+  role_name: srs_edge
+  author: Jan Koppe <post@jankoppe.de>
+  description: compile edge server and open rtmp firewall port
```
Comment (JanKoppe) [inline roles/srs_edge/meta/main.yaml]: should be "deploy SRS edge server".
- compilation is only done for the configuration file (from the template), not the actual SRS code.
- firewall ports are opened in other roles as well, but not mentioned in the description.

File: roles/srs_origin/meta/main.yaml [hunk 4]
```diff
@@ -1,3 +1,21 @@
 ---
+galaxy_info:
+  role_name: srs_origin
+  author: Jan Koppe <post@jankoppe.de>
+  description: compile origin server and open rtmp firewall port
```
Comment (JanKoppe) [inline roles/srs_origin/meta/main.yaml]: should be "deploy SRS origin server".
* compilation is only done for the configuration file (from the template), not the actual SRS code.
* firewall ports are opened in other roles as well, but not mentioned in the description.

**Review by JanKoppe — CHANGES_REQUESTED** [review]
Thank you very much! Lots of work done here :)

Overall it seems like you did not mention the focal Ubuntu release as a platform, which we are now using though. It's the super-new 20.04 LTS release.

And some nit-picking: "CHAOSWEST TV" seems weird. "Chaoswest TV" is a more friendly capitalization I think.

File: roles/srs_origin/meta/main.yaml (hunk 4 above)
Comment (DO1JLR) [reply roles/srs_origin/meta/main.yaml]: Thanks, it is updated

File: roles/srs_edge/meta/main.yaml (hunk 3 above)
Comment (DO1JLR) [reply roles/srs_edge/meta/main.yaml]: :heavy_check_mark: makes sense

File: roles/grafana/meta/main.yaml (hunk 2 above)
Comment (DO1JLR) [reply roles/grafana/meta/main.yaml]: updated to ``install grafana``

File: roles/docker/meta/main.yaml (hunk 1 above)
Comment (DO1JLR) [reply roles/docker/meta/main.yaml]: :heavy_check_mark: typo, sorry

**Comment by DO1JLR** [comment]
> Thank you very much! Lots of work done here :)

:heart: 

> Overall it seems like you did not mention the focal Ubuntu release as a platform, which we are now using though. It's the super-new 20.04 LTS release.

I didn't add this because the lint check at galaxy.ansible.com did not know this ubuntu version yet. But since this is not at galaxy and Ubuntu 20.4 is a LTS version again, there is no reason not to include it in the meta infos... so I just did that :wink:

> And some nit-picking: "CHAOSWEST TV" seems weird. "Chaoswest TV" is a more friendly capitalization I think.

good point. It is changed.

**Review by JanKoppe — APPROVED** [review]

**Pull request merged**